BorrowBooks/features/authentication/login_server.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once __DIR__ . '/../../repositories/UserRepository.php';
+
+use repositories\UsersRepository;
+
+session_start();
+
+$email = $_POST['email'];
+$password = $_POST['password'];
+
+function alert($message) {
+    echo "<script type='text/javascript'>alert('$message'); window.location.href = 'login.php';</script>";
```

Comment:
Would the window.location.href = 'login.php'; cause the page to reload leading to email and password fields resetting after wrong password for example?

Reply:
Yes, it refreshes the page, but there is no easy way to avoid this without refreshing. I tried using AJAX, but it didn't work as expected. I decided not to pursue this further since it would require more extensive research. If you have any ideas, I am open to suggestions.